diff --git a/template/src/webpack.config.js b/template/src/webpack.config.js
index 3075193276def91153bc5f330ccd0a8b7f88173b..9ef2d46aaacb924bccab19c4a3c7efd8108889d5 100644
--- a/template/src/webpack.config.js
+++ b/template/src/webpack.config.js
@@ -11,7 +11,7 @@
 
 /* webpack js bundler config file */
 const path = require('path');
-const { ESBuildMinifyPlugin } = require('esbuild-loader');
+const { EsbuildPlugin } = require('esbuild-loader');
 
 module.exports = {
   entry: path.resolve(__dirname, 'main.js'),
@@ -46,7 +46,7 @@ module.exports = {
   },
   optimization: {
     minimizer: [
-      new ESBuildMinifyPlugin({
+      new EsbuildPlugin({
         target: 'es2015',
       }),
     ],
